src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class Ball seems correctly implemented but make sure to include access modifiers for the class definition. By default, it will have package-private access, which may not be intended.